setup.py

```diff
@@ -381,7 +381,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 ]
 kubernetes = [
     'cryptography>=2.0.0',
-    'kubernetes>=3.0.0',
+    'kubernetes>=21.7.0',
```

Comment:
We should also update the code to delete references to classes that only exist in old versions of the API client.
See: https://github.com/apache/airflow/blob/f2039b4c9e15b514661d4facbd710791fe0a2ef4/airflow/providers/cncf/kubernetes/utils/pod_manager.py#L40-L45

Reply:
OK @mik-laj i made this change.  I could not find any other instances like this.  Any concerns with merging now?